package-lock.json

```diff
@@ -13548,11 +13548,22 @@
             }
         },
         "image-size": {
-            "version": "0.5.5",
-            "resolved": "https://registry.npmjs.org/image-size/-/image-size-0.5.5.tgz",
-            "integrity": "sha1-Cd/Uq50g4p6xw+gLiZA3jfnjy5w=",
-            "dev": true,
-            "optional": true
+            "version": "1.0.0",
```

Comment:
Any idea why this was updated, when package.json isn't updated?
Or did you forget to use `npm ci` instead of `npm i/npm install`